qemu/lib/iozone.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import os
+
+from avocado.core import exceptions
+from virttest import utils_misc
+from virttest import postprocess_iozone
+
+
+class Iozone(object):
+    """
+    Prepare iozone env, run iozone on guest, clean up iozone env.
+    """
+    def __init__(self, test, params, vm, session):
+        """
+        run setup of iozone test
+        """
+        self.params = params
+        self.session = session
+        self.test = test
+        self.results_path = ""
+        self.timeout = float(params.get("iozone_timeout", 240))
+        check_cmd = params.get("check_cmd")
```

Comment:
I think `check_cmd`, `dowmload_cmd` and 'setup_cmd' should be required param.
So use params.get["check_cmd"] would be better.

Then you don't need to check `if check_cmd:` and `if download_cmd and setup_cmd:` on line 29 and 44

Reply:
At present, in our test, only installation for linux guest is needed. For windows, we use iozone.exe in win_utils.iso. So "check_cmd/downlad_cmd/setup_cmd" will be defined only in linux cfgs, and windows guest will skip the check/installation/cleanup process.
  If we want to define windows iozone installation some day, we just need to define "check_cmd/download_cmd/setup_cmd" in our windows.cfg without no codes update in iozone.py, then all the check/installation/clean_up process will be run on windows guest either.

Reply:
OK, got it

Reply:
Are you running this script for both Linux and Windows?